cmd/neofs-adm/internal/modules/fschain/dump_names.go

```diff
@@ -70,7 +71,7 @@ func dumpNames(cmd *cobra.Command, _ []string) error {
 			return len(iParts) < len(jParts)
 		}
 		for k := len(iParts) - 1; k >= 0; k-- {
-			var c = strings.Compare(iParts[k], jParts[k])
+			var c = cmp.Compare(iParts[k], jParts[k])
```

Comment:
With 1.23+ `strings.Compare` is the same as `cmp.Compare` performance-wise. This can be omitted, less packages to import.

Reply:
dropped this commit